fiatconnect-api.md

```diff
@@ -2227,12 +2228,56 @@ A KYC schema containing personal data about a user, as well as documents such as
 	},
 	phoneNumber: `string`,
 	selfieDocument: `string`,
-	identificationDocument: `string`
+    identificationDocument: `string`
 }
 ```
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsDetailed`
+A more detailed Schema allowing the user to provide more information to work with CICO providers that require it.
+
+```
+{
+	firstName: `string`,
+	middleName?: `string`,
+	lastName: `string`,
+	dateOfBirth: {
+		day: `string`,
+		month: `string`,
+		year: `string`
+	},
+	address: {
+		address1: `string`,
+		address2?: `string`,
+		isoCountryCode: `string`,
+		isoRegionCode: `string`,
+		city: `string`,
+		postalCode?: `string`
+	},
+	phoneNumber: `string`,
+    email: `string`,
+	selfieDocument: `string`,
+    identificationDocumentType: `enum`,
+	identificationDocumentFront: `string`,
+    identificationDocumentBack?: `string` 
+}
+```
+
+The `phoneNumber` field is REQUIRED and MUST follow the formating of the [E.164 international standard](https://en.wikipedia.org/wiki/E.164). It MUST contain all parts of the phone number INCLUDING the Country Code.
+
+The `email` field MUST be a [valid email](https://en.wikipedia.org/wiki/Email_address#Syntax).
+
+The accepted documents can be found [here](https://en.wikipedia.org/wiki/Identity_document). The document should fit one of the `identificationDocumentType` field options:
+- `IDC`: State issued identity card
+- `PAS`: Passport
+- `DL`: Driving Liscense
+
+The `selfieDocument`, `identificationDocumentFront` and `identificationDocumentFront` fields should be base64 encoded binary blobs representing images.
+
+The `identificationDocumentBack` field is REQUIRED if the `identificationDocumentType` is `IDC` or `DL`. Otherwise, it is optional.
+
+
```

Comment:
You can add this section here to specify the enum I mentioned in another comment.

`````suggestion
##### 9.3.1.2.1 `IdentificationDocumentType`

`IdentificationDocumentType` is used to represent the type of document being submitted with a `PersonalDataAndDocumentsDetailed` KYC schema.

```
[
  `IDC`,
  `PAS`,
  `DL`
]
```

The enum values represent the following forms of identification:
* `IDC`: State-issued identity card
* `PAS`: Passport
* `DL`: Driver's License
`````